tests/nn/data_parallel/test_fsdp_regnet.py

```diff
@@ -54,7 +45,16 @@ def forward(self, x):
     # TODO (Min): check DDP equivalency.
```

Comment:
having been burnt a little by that, I would recommend not waiting too long for that part

Reply:
definitely. see my plan below.